pipeline-elastic/README.adoc

```diff
@@ -58,3 +58,35 @@ echo-hello-world-task-run-pod-f4ee3b   0/1       Completed   0          35s
 Access to the http://localhost:30601/app/infra#/logs?logFilter=(expression:'kubernetes.labels.build.knative.dev%2FbuildName:%20%22echo-hello-world-task-run%22',kind:kuery)&logPosition=(position:(tiebreaker:3620,time:1547642338000),streamLive:!t)&_g=()[Pod Logs] in Kibana
 
 image::kibana-pod-logs.png[width=90%]
+
+
+
+== Kaniko Hello World - Dockerhub
+
+This is an hello world to build and push a Docker image to Dockerhub
+
+=== Prerequisite
+
+1. You need your `~/.docker/config.json` file configured
```

Comment:
This could say something like
```
Your docker daemon needs the ability to push to docker hub. For example using `docker login`, documentation: https://docs.docker.com/engine/reference/commandline/login/
```

Reply:
Okay I am getting further into the tutorial and see how the file is being mounted, I'll complete the tutorial and dig in a bit more to understand how this is being used.